bin/getBUFR

```diff
@@ -243,6 +243,7 @@ if __name__ == '__main__':
 						continue
 					elif min_data_pos_result is False:
 						failed_min_data_pos.append(stid)
+						positions = fetch_old_positions(df1, stid, args.time_limit, positions)
```

Comment:
Let me try to push some code, I think I have an idea that will work. Hopefully just 30 minutes here.

Reply:
> The problem with this is that we are simply creating the positions dict (which will be written to file, and used in the csv), however we still hit the continue so we will skip this station and never run getBUFR. Even if we did run getBUFR it would still be using s1_current which has the original unmodified positions.

That was intended. It is not urgent to start sending BUFR files with extrapolated elevations to DMI. I would like for example to clean up the available GPS data, so that the extrapolation is more robust, before we send the extrapolation result to DMI. At the moment, I'm just trying to send a best guess to AWS_positions.csv .